### PR TITLE
add tracking limit order + update ux limit order

### PR DESCRIPTION
**Pull request by nguyenhoaidanh**
- add tracking limit order
- fix bug can not verify hardware wallet
- update ux limit order

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -86,24 +84,6 @@ if (ENV_LEVEL > ENV_TYPE.LOCAL) {
 
 AOS.init()
 
-// customer support widget
-window.zESettings = {
```
Comment (namgold) [inline src/index.tsx:90]: remove in `react-app-env.d.ts` too

File: src/components/swapv2/LimitOrder/useBaseTradeInfo.ts [hunk 2]
```diff
@@ -36,25 +33,27 @@ export default function useBaseTradeInfo(currencyIn: Currency | undefined, curre
...
       },
-    })
-    const toCurrencyAmount = function (value: string, currency: Currency): CurrencyAmount<Currency> {
-      return TokenAmount.fromRawAmount(currency, JSBI.BigInt(value))
-    }
-    const { outputAmount, amountInUsd, amountOutUsd, routerAddress } = data
-    const amountOut = toCurrencyAmount(outputAmount, currencyOut)
+    }).then(data => data.json())
+
+    const { outputAmount, amountInUsd } = data
```
Comment (namgold) [inline src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:50]: May crash. Should wrap a try catch?

Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/useBaseTradeInfo.ts:50]: no need, already added try catch in useSWR

File: src/index.tsx [hunk 3]
```diff
@@ -86,24 +84,6 @@ if (ENV_LEVEL > ENV_TYPE.LOCAL) {
...
-      },
-    },
-    zIndex: Z_INDEXS.ICON_SUPPORT,
-  },
-}
-const script = document.createElement('script')
-script.src = 'https://static.zdassets.com/ekr/snippet.js?key=a73faacd-ba50-493a-8bf5-4b6878035346'
-script.id = 'ze-snippet'
-document.body.appendChild(script)
-
```
Comment (namgold) [inline src/index.tsx:106]: This should be a separated PR. This need go to main soon, we need it too

Comment (nguyenhoaidanh) [reply src/index.tsx:106]: I will merge this pr now after your approval. 🙏

Comment (namgold) [reply src/index.tsx:106]: ah sorry I though that we are merging to another branch :))

File: src/components/swapv2/LimitOrder/LimitOrderForm.tsx [hunk 4]
```diff
@@ -435,7 +458,15 @@ const LimitOrderForm = function LimitOrderForm({
...
     }))
     const signature = await library.getSigner().signMessage(ethers.utils.arrayify(orderHash))
-    return { signature, orderHash }
+
+    const bytes = ethers.utils.arrayify(signature)
+    const lastByte = bytes[64]
+    if (lastByte === 0 || lastByte === 1) {
+      // to support hardware wallet https://ethereum.stackexchange.com/a/113727
+      bytes[64] += 27
+    }
```
Comment (namgold) [inline src/components/swapv2/LimitOrder/LimitOrderForm.tsx:467]: tested?

Comment (nguyenhoaidanh) [reply src/components/swapv2/LimitOrder/LimitOrderForm.tsx:467]: yes, QC test sấp mặt.

**Review by namgold — APPROVED** [review]

**Pull request merged**